darwin/exporter/formats/yolo.py

```diff
@@ -0,0 +1,90 @@
+from functools import partial
+from pathlib import Path
+from typing import Any, Callable, Dict, Iterable, List, Tuple, Union
+
+from rich.console import Console
+from rich.theme import Theme
+
+from darwin.datatypes import (
+    Annotation,
+    AnnotationFile,
+    ConversionError,
+    VideoAnnotation,
+    YoloAnnotation,
+)
+
+
+def export(annotation_files: Iterable[AnnotationFile], output_dir: Path) -> None:
+    """
+    Exports the given ``AnnotationFile``s into the yolo format inside of the given ``output_dir``.
+
+    Parameters
+    ----------
+    annotation_files : Iterator[dt.AnnotationFile]
+        The ``AnnotationFile``s to be exported.
+    output_dir : Path
+        The folder where the new coco file will be.
+    """
+    console = Console(theme=_console_theme())
+
+    output_dir.mkdir(parents=True, exist_ok=True)
+    for annotation_file in annotation_files:
+        output_file_path = (output_dir / annotation_file.filename).with_suffix(".txt")
+        errors, yolo_annotations = _convert_file(annotation_file)
```

Comment:
So if a file cannot be converted, we're not blocking the execution. This obviously has some useful properties, but I'm not sure it's consistent with the other exporters.

Reply:
Currently, there is no consistent behaviour. Some exports, like coco, have a full dataset inside, while other, have an export file for each image. I could with the "All or Nothing" approach here, but I think it is better for the customers to have something working, than nothing. At least this is the idea I have from Rafal Z and James.